### ISSUE TITLE
packages are not backed up

### ISSUE THREAD

**Issue by alirezaghey** [sameersbn/docker-gitlab#2825]
Hey folks,

I'm on `version 13.11.4` and currently migrating to a package based instance. Everything works fine with the backup and restore process except for the `packages`. I haven't skipped them in the setting. I have `GITLAB_BACKUP_SKIP=builds,tar`.
My shared folder looks like this:
```console
[root@gitlab-server shared]# ls -al /opt/gitlab/gitlab/shared
total 4
drwxr-xr-x.  8 centos centos   98 Feb  1  2022 .
drwxr-xr-x. 11 centos centos  131 Sep  8 08:15 ..
drwxr-xr-x. 46 centos centos 4096 Aug 21 13:26 artifacts
drwxr-xr-x.  2 centos centos    6 May 20  2021 lfs-objects
drwxr-xr-x. 20 centos centos  187 Jul  5 14:23 packages
drwxr-xr-x.  2 centos centos    6 May 20  2021 pages
drwxr-xr-x.  3 centos centos   20 Mar 15  2022 registry
drwxr-xr-x.  5 centos centos   53 Aug 10  2021 tmp
```
But my backups look like the following:
```console
[root@gitlab-server shared]# ls -al /opt/gitlab/gitlab/backups
total 5711564
drwxr-xr-x.  4 centos centos        173 Sep  8 02:05 .
drwxr-xr-x. 11 centos centos        131 Sep  8 08:15 ..
-rw-------.  1 centos centos 1781649677 Sep  8 02:05 artifacts.tar.gz
-rw-rw-r--.  1 centos centos        193 Sep  8 02:09 backup_information.yml
drwxrwxr-x.  2 centos centos         29 Sep  8 02:01 db
-rw-------.  1 centos centos        146 Sep  8 02:05 lfs.tar.gz
-rw-------.  1 centos centos        146 Sep  8 02:05 pages.tar.gz
-rw-------.  1 centos centos 4066756562 Sep  8 02:09 registry.tar.gz
drwx------.  3 centos centos         21 Sep  8 02:01 repositories
-rw-------.  1 centos centos     213137 Sep  8 02:02 uploads.tar.gz
```
As you can see there is no sign of packages.
I also unpacked all the backed up files and searched in them for the package files and they were not there.
I'd like to mention that all checks on the migrated server seem normal. The package files show up in the migrated server but when I try to download them I get a `500` error. Upon investigating the logs, I see that the files are missing in `shared` folder. In fact, there are no files at all in the migrated server in `shared/packages`.

Appreciate any feedback! Thank you!

**Comment by kkimurak** [comment]
As you are using 13.11.4, packages are not contained in backup. Please backup/restore them individually.
This have been fixed on 14.7 release. See https://about.gitlab.com/releases/2022/01/22/gitlab-14-7-released/#backup-and-restore-supports-package-registry-files  

Also, this issue is duplicate of #2323 .

**Comment by alirezaghey** [comment]
Thanks mate, much appreciated!
